tests/shipping/models/Pickup - No number.model.js

```diff
@@ -0,0 +1,53 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import {
+  fillEmail,
+  getRandomEmail,
+  fillProfile,
+} from '../../../utils/profile-actions'
+import {
+  goToPayment,
+  fillPostalCodeOmnishipping,
+  choosePickup,
+} from '../../../utils/shipping-actions'
+import {
+  completePurchase,
+  payWithCreditCard,
+} from '../../../utils/payment-actions'
+import { SKUS, PICKUP_TEXT } from '../../../utils/constants'
+
+export default function test(account) {
+  describe(`Pickup - Address with no number - ${account}`, () => {
```

Comment:
Could you make the `describe` have the same text as the filename? I don't remember if that's a problem, but I think it makes it easier to search for it later on

Reply:
(or the other way round)